core/include/gnuradio-4.0/CircularBuffer.hpp

```diff
@@ -654,8 +665,8 @@ class CircularBuffer
     [[nodiscard]] constexpr const T* data() const noexcept { return _internalSpan.data(); }
     const T& operator [](std::size_t i) const noexcept  {return _internalSpan[i]; }
     const T& operator [](std::size_t i) noexcept { return _internalSpan[i]; }
-    operator const std::span<const T>&() const noexcept { return _internalSpan; }
-    operator std::span<const T>&() noexcept { return _internalSpan; }
+    explicit(false) operator const std::span<const T>&() const noexcept { return _internalSpan; }
+    explicit(false) operator std::span<const T>&() noexcept { return _internalSpan; }
```

Comment:
Code analyzers will complain on implicit conversion operators and ask you  to make them expclicit. We have that at multiple occasions, the original idea was that this should suppress these warnings (I think it does for nodiscard(false)). While it does not actually suppress the warning, it will still hopefully deter someone going through all sonartype issues to just make it explicit which would be a breaking change.